vmdb/db/fixtures/miq_product_features.yml

```diff
@@ -3479,4 +3479,192 @@
+      :feature_type: view
+      :identifier: ems_container_show_list
+    - :name: Show
+      :description: Display Individual Containers Providers
+      :feature_type: view
+      :identifier: ems_container_show
+  - :name: Modify
+    :description: Modify Containers Providers
+    :feature_type: admin
+    :identifier: ems_container_admin
+    :children:
+    - :name: Remove
+      :description: Remove Containers Providers from the VMDB
+      :feature_type: admin
+      :identifier: ems_container_delete
+    - :name: Edit
+      :description: Edit a Containers Provider
+      :feature_type: admin
+      :identifier: ems_container_edit
+    - :name: Add
+      :description: Add a Containers Provider
+      :feature_type: admin
+      :identifier: ems_container_new
+
+# ContainerGroup
+- :name: Container Groups
+  :description: Everything under Container Groups
+  :feature_type: node
+  :identifier: container_group
+  :children:
+  - :name: View
+    :description: View Container Groups
+    :feature_type: view
+    :identifier: container_group_view
+    :children:
+    - :name: List
+      :description: Display Lists of Container Groups
+      :feature_type: view
+      :identifier: container_group_show_list
+    - :name: Show
+      :description: Display Individual Container Groups
+      :feature_type: view
+      :identifier: container_group_show
+  - :name: Modify
+    :description: Modify Container Groups
+    :feature_type: admin
+    :identifier: container_group_admin
+    :children:
+    - :name: Remove
+      :description: Remove Container Groups from the VMDB
+      :feature_type: admin
+      :identifier: container_group_delete
+    - :name: Edit
+      :description: Edit a Container Groups
+      :feature_type: admin
+      :identifier: container_group_edit
+    - :name: Add
+      :description: Add a Container Groups
+      :feature_type: admin
+      :identifier: container_group_new
+
+# ContainerNode
+- :name: Nodes
+  :description: Everything under Container Nodes
+  :feature_type: node
+  :identifier: container_node
+  :children:
+  - :name: View
+    :description: View Container Nodes
+    :feature_type: view
+    :identifier: container_node_view
+    :children:
+    - :name: List
+      :description: Display Lists of Container Nodes
+      :feature_type: view
+      :identifier: container_node_show_list
+    - :name: Show
+      :description: Display Individual Container Nodes
+      :feature_type: view
+      :identifier: container_node_show
+  - :name: Modify
+    :description: Modify Container Nodes
+    :feature_type: admin
+    :identifier: container_node_admin
+    :children:
+    - :name: Remove
+      :description: Remove Container Nodes from the VMDB
+      :feature_type: admin
+      :identifier: container_node_delete
+    - :name: Edit
+      :description: Edit a Container Nodes
+      :feature_type: admin
+      :identifier: container_node_edit
+    - :name: Add
```

Comment:
I'm not sure we are currently ready or want to support add/remove/modify of container nodes via manageiq so we might not want to expose that.
@simon3z  thoughts?